packages/cms-lib/oauth.js

```diff
@@ -31,6 +32,43 @@ const getOauthManager = (portalId, portalConfig) => {
   return oauthManagers.get(portalId);
 };
 
+const setupOauth = (portalId, portalConfig) => {
```

Comment:
I just noticed this, but I think that this could have used OAuth2Manager.fromConfig`.